client/rpc/rpc_sender.go

```diff
@@ -78,8 +79,8 @@ func NewRPCSender(server string, context *base.Context) (*RPCSender, error) {
 // and been executed successfully. We retry here to eventually get
 // through with the same client command ID and be given the cached
 // response.
-func (s *RPCSender) Send(_ context.Context, call Call) {
-	retryOpts := HTTPRetryOptions
+func (s *Sender) Send(_ context.Context, call client.Call) {
+	retryOpts := client.HTTPRetryOptions
```

Comment:
Should probably rename `client.HTTPRetryOptions` to `client.RetryOptions`. Let's leave that to a subsequent change.